components/navbar/navbar-top.vue

```diff
@@ -1,11 +1,70 @@
 <template>
-  <nav class="mb-5">
-    navbar top
+  <nav class="mb-5 flex justify-between">
+    <div class="flex items-center">
+      <a
+        class="p-2 icon"
+        href="https://app.slack.com/client/TDJE5CKHP/CDJV3SA11"
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        <font-awesome-icon
+          :icon="['fab', 'slack']"
+          class="fill-current text-mtl-primary w-4 h-4"
+        />
+      </a>
+      <a
+        class="p-2 icon"
+        href="https://www.linkedin.com/groups/12289604/"
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        <font-awesome-icon
+          :icon="['fab', 'linkedin']"
+          class="fill-current text-mtl-primary w-4 h-4"
+        />
+      </a>
+      <a
+        class="p-2 icon"
+        href="https://github.com/vuemontreal/vuemontreal"
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        <font-awesome-icon
+          :icon="['fab', 'github']"
+          class="fill-current text-mtl-primary w-4 h-4"
+        />
+      </a>
+    </div>
+    <div class="flex items-center">
+      <input
+        v-model="search"
+        type="text"
+        class="border border-black border-solid rounded-full pl-4"
+        placeholder="search"
+      />
+    </div>
   </nav>
 </template>
 
 <script>
 export default {
-  name: 'NavBarTop'
+  name: 'NavBarTop',
+  data() {
+    return {
+      search: ''
+    }
+  }
 }
 </script>
+
+<style>
+.icon {
+  left: -10px;
+  position: relative;
+}
+
+.icon svg {
+  height: 1.5rem !important;
```

Comment:
is that a css-lib-overide??

Reply:
yep, couldn't find a way to do this with Tailwind as the svg size is coming inline from FontAwesome

Reply:
if it's coming from fontawesome yeah you have no choice because it's about css selector length